tutordiscovery/__about__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "16.0.0"
+__version__ = "16.0.1"
```

Comment:
Please don't bump the version number in this PR. Instead, push a separate commit that includes just the version number change (see git history).

Reply:
ok sure